pkg/manager/member/startscript/v1/render_script.go

```diff
@@ -70,9 +71,55 @@ func RenderPDStartScript(tc *v1alpha1.TidbCluster) (string, error) {
 	if tc.Spec.PD.StartUpScriptVersion == "v1" {
 		model.CheckDomainScript = checkDNSV1
 	}
+
+	pdStartSubScript := ``
+	mode := ""
+	if tc.Spec.PD.Mode == "ms" {
+		mode = "api"
+	}
```

Comment:
if the mode in PD is `api`, could we also use `api` instead of `ms` in Operator?

Reply:
- ms indicates pd is in micro service model
- api means we need a pd server to supply `api` service

Reply:
OK